### ISSUE TITLE
Android installation is broken

### ISSUE THREAD

**Issue by tothvoj-gl** [Tealium/tealium-react-native#21]
When you run `npm install tealium-react-native` and `react-native link` the android version of the app can not be build. Few things are missing:

1. The code for declaring "http://maven.tealiumiq.com/android/releases" as a maven URL for all projects is commented out in the SDK's gradle file. Therefore we need it to declare in our app's gradle file.
2. `import com.tealiumreactnative.TealiumPackage;` is missing from the Application, needs to be added manually.

SDK version: 1.0.4
RN version: 0.59.2

**Comment by jameskeith** [comment]
Thanks @tothvoj-gl for letting us know. 
I've run through a clean installation at my end, creating a blank test application in React using the following 
```
react-native init TestApp
cd TestApp
npm install tealium-react-native
react-native link
```
This seems to work just fine. All the gradle/maven dependencies along with the appropriate `import` statements are added.
I suspect that this is down to `react-native link` as I understand it's not the most reliable tools for non-vanilla applications I'm afraid.
 - Have you tried linking just the `tealium-react-native` module by itself? `react-native link tealium-react-native`

SDK version: 1.0.4
react-native-cli: 2.0.1
react-native: 0.59.9

**Comment by tothvoj-gl** [comment]
Yes, I used the react-native link tealium-react-native command. If you cannot reproduce it, than it is likely our project configuration which caused the issue.

**Comment by jameskeith** [comment]
@tothvoj-gl 
The latest release - 1.0.7 - should now support Autolinking in React Native 0.60+
This should mean it will automatically take care of the `import` and `new TealiumPackage()` statements in your Application class, and we've therefore removed the previous scripts that did this work.
You will need to add a reference to our maven repo as outlined [here](https://docs.tealium.com/platforms/android/install/#maven-install) on our Android documentation - only the part in your project's root build.gradle.

let me know if you still have any problems

**Comment by jameskeith** [comment]
resolved in 1.0.7 - closing as no further updates provided on this issue.
Please let us know if you still have this problem.
